components/brave_wallet/browser/password_encryptor.h

```diff
@@ -22,11 +23,14 @@ namespace brave_wallet {
 inline constexpr size_t kEncryptorNonceSize = 12;
 inline constexpr size_t kEncryptorSaltSize = 32;
 inline constexpr int kPbkdf2Iterations = 310000;
-inline constexpr int kPbkdf2KeySize = 256;
+inline constexpr int kPbkdf2KeySize = 32;
```

Comment:
Why this was changed?

Reply:
Let's be consistent and avoid mixing sizes in bytes and bits